geometry_msgs/Pose2D center
float64 area
float64 size_x
float64 size_y
string name